torch/nn/quantized/modules/linear.py

```diff
@@ -91,18 +91,20 @@ class Linear(NNLinear):
     __constants__ = ['bias', 'in_features', 'out_features']
 
     def __init__(self, in_features, out_features, bias=True):
-        assert bias, 'nobias is not supported in Quantized Linear module yet'
         super(Linear, self).__init__(in_features, out_features, bias)
+        if bias:
+            del self.bias
+            qbias = torch._empty_affine_quantized(
+                [out_features], scale=1, zero_point=0, dtype=torch.qint32)
+            self.register_buffer('bias', qbias)
+        else:
+            self.register_buffer('bias', None)
         del self.weight
-        del self.bias
         qweight = torch._empty_affine_quantized(
             [out_features, in_features], scale=1, zero_point=0,
             dtype=torch.qint8)
-        qbias = torch._empty_affine_quantized(
-            [out_features], scale=1, zero_point=0, dtype=torch.qint32)
         self.register_buffer('_packed_weight',
                              torch.ops.quantized.fbgemm_linear_prepack(qweight))
-        self.register_buffer('bias', qbias)
         self.register_buffer('out_scale', torch.Tensor([1]))
```

Comment:
could you change this to use
`torch.tensor([1], dtype=torch.double)`
and
`torch.tensor([0], dtype=torch.float)`

Reply:
Updated. Thanks @jerryzh168 !